### ISSUE TITLE
TOC Does not generate in a file produced from a powershell script. 

### ISSUE THREAD

**Issue by smchargue** [thlorenz/doctoc#210]
TLDR - DOCTOC would not create/update TOC for  PowerShell Generated Markdown files because of the default encoding.

I'm using this utility to auto-gen a TOC into markdown files that are generated by powershell.   It was failing to create the TOC but reported no errors.  After much trial and error, i created this simple example that reproduced the error:

``` powershell
"# Header `n" | Out-File -FilePath .\simple.md 
"## Sub Header B`n" | Out-File -FilePath .\simple.md -Append 
"### Sub Header B1`n" | Out-File -FilePath .\simple.md -Append 
"#### Sub Header B1a`n" | Out-File -FilePath .\simple.md -Append 
```
As it turns out, I needed to force the Encoding.  I used Ascii and it worked (did not try others).  According to ms docs the default is utf8NoBOM

so this worked 
``` powershell
"# Header `n" | Out-File -FilePath .\simple.md -Encoding ASCII
"## Sub Header B`n" | Out-File -FilePath .\simple.md -Append -Encoding ASCII
"### Sub Header B1`n" | Out-File -FilePath .\simple.md -Append -Encoding ASCII
"#### Sub Header B1a`n" | Out-File -FilePath .\simple.md -Append -Encoding ASCII
```

Probably a real edge case and no fix required to the utility. Just putting it out there if it saves someone else the trouble.
